packages/td-bank-integration/src/td-online-mart-payment-strategy.ts

```diff
@@ -147,4 +154,29 @@ export default class TDOnlineMartPaymentStrategy implements PaymentStrategy {
 
         return this.tdOnlineMartClient;
     }
+
+    private errorHandler(error: any) {
+        if (!error?.message) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentToken);
```

Comment:
if you are using `MissingPaymentToken` error as default message, maybe better to rename this method to something like `tokenizationErrorHandler` or `throwTokenizationError` ?